client/checks.go

```diff
@@ -100,3 +103,29 @@ func (client *Client) Checks(opts *ChecksOptions) ([]*CheckInfo, error) {
 	}
 	return checks, nil
 }
+
+type CheckPayload struct {
+	Action string
+	Check  string
+}
+
+// RunCheck runs a specific health check immediately and return the status.
+func (client *Client) RunCheck(opts *CheckPayload) (string, error) {
+	body, err := json.Marshal(opts)
```

Comment:
Have a look at how the other actions work. The `RunCheckOptions` struct should be separate from the `runCheckPayload` struct that's marshalled for the API.